backend/src/utils/Mailer.ts

```diff
@@ -28,7 +28,7 @@ export async function sendResetPasswordEmail(username: string, email: string, co
                     <p>Привет, ${username}!</p>
                     <p>Чтобы сбросить пароль, пройдите по этой ссылке: ${(siteConfig.http ? 'http://' : 'https://') + siteConfig.domain}/forgot-password/${code}</p>
                     <p>Ссылка активна только 3 часа</p>
-                    <p>Если ничего не получается, можете написать в <a href="https://t.me/orbitar_bot">телеграм боту</a></p>
+                    <p>Если ничего не получается, можете написать нам в <a href="https://discord.gg/j4tevpV9">Дискорд</a></p>
```

Comment:
Not sure if this is a good idea to post the discord invite on the (semi-) public page. Of course the invite can be invalidated, but still.

Reply:
Why not create and use a separate email, like support@orbitar.space ?